basic-integration/surfly-integration-with-tag-manager.md

```diff
@@ -0,0 +1,177 @@
+<a href="https://www.surfly.com/">![logo](../images/logosmall.png)</a>
+#Surfly integration with Tag Manager
+
+Tag management systems are mostly used to inject Javascript into websites for marketing purposes. But with most tag managers it is possible to inject any type of html into your website, and not only scripts. You can add text, buttons, etc. and even implement Surfly through a tag management system.
+
+In this chapter we'll show you how to add a basic Surfly implementation to your website using Google Tag Manager.
+
+####The Features
+
+Within this example we make use of the [session-id approach](../tutorial/advanced-integration.md), which gives you the following use-case:
+
+- The user is already chatting with an agent, and is directed to the "Get Help"-button to start a Surfly co-browsing session
+
+![get help](../images/get-help-button.png)
+
+- After the user clicks the "Get-Help"-button, a Surfly session is initiated, and the button now shows the session-id (which is by default a four-number code)
+- The user can now pass the session-id to the agent. The agent then uses the session-id to identify the correct session from the queue in his/her admin panel
+
+![chat box](../images/olark-chat-box.png)
+
+- The agent joins the session, and the session now becomes visible to both users
+
+####How it works
+
+With Google Tag Manager you are able to inject HTML into your website. As long as you put it in between the correct tags, you can add divs, css, javascript, etc., to your website's page(s). The tag you inject through GTM can even contain more than one script, and even scripts of a different nature.
+
+In short, you add a tag to your website as follows:
+
+- Register at https://www.google.nl/tagmanager/
+- Log in, and create an account for your website
+- Add an account-name and a container (which is the url) and set the destination
+- You get two code snippets, follow the instructions on how to add these to your website
+- You can create a new tag on the “workspace” console, or click on “tags” >> “new”
+- Add a name
+- Click on “tag configuration ” select "custom tag", then add the code snippet. You can add anything as long as it’s in HTML
+- Enable document.write
+- Add a trigger. Here you can select the pages on which you want the tag to be implemented
+- When you’re ready to review, click the carret-down next to “publish” to create a new version
+- Go to “versions” >> “actions” >> “review” to test the tag's functionality on your website
+- If everything is working properlyl, go to “versions” >> “actions” >> “publish” to make your tag go live
+
+{% em color="#ffffe0" %} For more info on Google Tag Manager, please visit the website https://www.google.com/analytics/tag-manager {% endem %}
+
+####The Tag
+
+Below, you'll find an example of how a tag, using the session-id approach, would look like. The example also includes the option of adding a third-party chat solution to your website. In this case we've used Olark.
```

Comment:
Perhaps include a link to Olark website? and maybe mention "existing chat solution" in the title? what do you think?